### ISSUE TITLE
Basic CSS: Use CSS Selectors to Style Elements challenge doesnt accept rigth color style

### ISSUE THREAD

**Issue by alivemary** [freeCodeCamp/freeCodeCamp#14064]
Challenge [use-css-selectors-to-style-elements](https://beta.freecodecamp.com/en/challenges/basic-css/use-css-selectors-to-style-elements) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 6.1; WOW64; rv:52.0) Gecko/20100101 Firefox/52.0</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

![Issue](https://alivemary.github.io/img/issue.jpg)

```html

<style>
  h2 {color: blue;}
</style>
<h2>CatPhotoApp</h2>
<main>
  <p>Click here to view more <a href="#">cat photos</a>.</p>
  
  <a href="#"><img src="https://bit.ly/fcc-relaxing-cat" alt="A cute orange cat lying on its back."></a>
  
  <div>
    <p>Things cats love:</p>
    <ul>
      <li>cat nip</li>
      <li>laser pointers</li>
      <li>lasagna</li>
    </ul>
    <p>Top 3 things cats hate:</p>
    <ol>
      <li>flea treatment</li>
      <li>thunder</li>
      <li>other cats</li>
    </ol>
  </div>
  
  <form action="/submit-cat-photo">
    <label><input type="radio" name="indoor-outdoor" checked> Indoor</label>
    <label><input type="radio" name="indoor-outdoor"> Outdoor</label><br>
    <label><input type="checkbox" name="personality" checked> Loving</label>
    <label><input type="checkbox" name="personality"> Lazy</label>
    <label><input type="checkbox" name="personality"> Energetic</label><br>
    <input type="text" placeholder="cat photo URL" required>
    <button type="submit">Submit</button>
  </form>
</main>

```

**Comment by JoolsMcFly** [comment]
Indeed. I'll see what I can do.
Stay tuned.

**Comment by marekjedlinski** [comment]
The same bug seems to apply to other challenges which test applying colors, e.g. https://beta.freecodecamp.com/en/challenges/basic-css/use-a-css-class-to-style-an-element and https://beta.freecodecamp.com/en/challenges/basic-css/style-multiple-elements-with-a-css-class

In the latter, the test for class ("Your h2 element should have the class red-text.") passes, but the test for redness ("Your h2 element should be red.") fails. The result is the same regardless of how the color is specified (name, hex etc.)

**Comment by hay-dee** [comment]
@alivemary The beta version of freeCodeCamp currently faces an issue the .css() jQuery method ,used to do the testing, on mozilla firefox. The issue is currently being addressed at #10749. This issue might also be a duplicate of #12677.

**Comment by systimotic** [comment]
Hi! Thanks for reporting and looking into this issue everyone!

The problem is exactly as described by @hay-dee (excellent work 👍)
I'm closing this as a duplicate of #10749. Happy coding!
